homeassistant/components/launch_library/translations/en.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This file seems outdated, please run `python -m script.translations develop`

Reply:
Done